pkg/api/plugin.go

```diff
@@ -119,4 +120,7 @@ type Plugin interface {
 
 	// GetControlPlanePods fetches a consolidated list of the control plane pods in the cluster
 	GetControlPlanePods(ctx context.Context, oc *OpenShiftManagedCluster) ([]byte, error)
+
+	// ForceUpdate forces an update to a cluster
+	ForceUpdate(ctx context.Context, cs *OpenShiftManagedCluster, deployer DeployFn, template *plugin.Config) *PluginError
```

Comment:
Let's prepend "Geneva" to all the Geneva Action functions

Reply:
https://github.com/openshift/openshift-azure/pull/1138